README.md

```diff
@@ -1,2 +1,30 @@
 # cr-pulse-interpolator
-Full waveform interpolation for air shower simulations
+Full electric-field waveform interpolation for air shower simulations
+
+For elaborate information on the methods and their performance, see  
+Corstanje et al. (2023), arXiv xxxx.xxxxx, doi xx.xxxx,  
```

Comment:
Don't forget to update this after arxiv submission is through.

Reply:
Sure